tests/Service/IncentiveImportServiceTest.php

```diff
@@ -39,4 +40,32 @@ public function emailDataProvider()
             ['pmi-ops.org@ops-pmi.org', false]
         ];
     }
+
+    /**
+     * @dataProvider csvFileDataProvider
+     */
+    public function testExtractCsvFileData($fileName, $isValid)
+    {
+        $form = static::$container->get('form.factory')->create(IncentiveImportFormType::class, null, ['csrf_protection' => false]);
+        $form->submit([
+            'incentive_csv' => $this->createUploadedFile($fileName)
+        ]);
+        $file = $form['incentive_csv']->getData();
+        $this->service->extractCsvFileData($file, $form);
+        $this->assertEquals($form->isValid(), $isValid);
```

Comment:
Could you also check the results of the `extractCsvFileData` method when it is valid? At least a check for the expected number of rows?